src/ipc-handlers.ts

```diff
@@ -220,11 +220,14 @@ export async function updateSite(
 	return mergeSiteDetailsWithRunningDetails( userData.sites );
 }
 
+type DataConnectWpcomSite = { sites: SyncSite[]; localSiteId: string };
+
 export async function connectWpcomSite(
 	event: IpcMainInvokeEvent,
-	sites: SyncSite[],
-	localSiteId: string
+	data: DataConnectWpcomSite | DataConnectWpcomSite[]
```

Comment:
I was woring about readability of suing this function, from my point of view is more readable when you do:
`connectWpcomSite( site )` instead of `connectWpcomSite( [ site ] )`. And to support it we need just one simple line of code `const list = Array.isArray( data ) ? data : [ data ];`

Reply:
I actually agree with @wojtekn that always passing either a single site or an array of sites makes more sense

Reply:
I don't have any concerns regarding it, so let's go with the suggestion, I will revert this code 👌